moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -632,18 +661,20 @@ void JogCalcs::enforceSRDFAccelVelLimits(Eigen::ArrayXd& delta_theta)
         acceleration_limit = bounds.max_acceleration_;
       }
 
-      // Apply acceleration bounds
-      if (clip_acceleration)
+      // Find relative change
+      if (clip_acceleration && delta_theta(joint_delta_index) != 0)
       {
         // accel = (vel - vel_prev) / delta_t = ((delta_theta / delta_t) - vel_prev) / delta_t
         // --> delta_theta = (accel * delta_t _ + vel_prev) * delta_t
         const double relative_change =
```

Comment:
Shouldn't we always take the absolute value here?

Reply:
I thought about this a bit, and decided that it was possible in extreme/fringe cases for the relative change to be negative (e.g. a positive delta_theta, but the previous velocity was very negative would result in a needed negative delta_theta to maintain acceleration limits, and means negative relative change). I don't think this actually happens in real life, but figured to guarantee accel limit enforcement, we need to allow the negative relative change here